content/kotlin/concepts/arrays/arrays.md

```diff
@@ -0,0 +1,143 @@
+---
+Title: 'Arrays'
+Description: 'An array stores multiple items of the same data-type, such as an integer or string, under a single variable name.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Data Types'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+An **array** in Kotlin is a collection of items of same data type stored at contiguous memory locations. This makes it easier to calculate the position of each element by simply adding an offset to a base value, i.e., the memory location of the first element of the array (generally denoted by the name of the array). The base value is index 0 and the difference between the two indexes is the offset.
+
```

Comment:
```suggestion

## Syntax

```

After this, let's add a new \```pseudo code snippet that looks like the following:

\`\`\`pseudo
// Array class instance
arrayOne = arrayOf(values)
arrayTwo = Array(size) iteratorFunction

// Array of primitive types
arrayThree: TypeArray = typeArrayOf(values)
arrayFour = TypeArray(size) iteratorFunction
\`\`\`

Reply:
@Dusch4593 check if this is right. This is what you wanted?